README.rst

```diff
@@ -15,60 +16,75 @@ Django GUID
     :target: https://django-guid.readthedocs.io/en/latest/?badge=latest
 
 
-Django GUID stores a GUID to an object, making it accessible by using the ID of the current thread.
```

Comment:
Informing the user on how something actually works under the hood is to me very beneficial and should always be done. It's the best way of getting contributions and ensuring the users don't feel the package is just magic. No matter what, I want this to either be in some `FAQ` in the docs, or kept in the README.

Reply:
How about adding this in before "Resources"?:

"For an in-depth explanation of how the package functions, see the docs_" and link to a page in the docs specifically for functionality?

Reply:
Or we could add a full explanation to the top of the index - I just dont think it should be 100% functional - think it's equally important to highlight the purpose of the package

Reply:
I totally agree! I'd like a short description on how somewhere though. 

Maybe just append to the top summary?

>Django GUID attaches a unique correlation ID to all your log outputs for every requests you handle. This means that every error now has an ID connecting it to all other relevant logs, making debugging simple.
>
> Django-guid stores a GUID to an object, making it accessible by using the ID of the current thread. This makes integrations possible, as the ID can be returned as a header(built in setting) or forwarded manually to other systems(built in API), making it possible to extend the reach of correlation IDs to whole systems.